centreon/www/class/centreon-clapi/centreonAPI.class.php

```diff
@@ -1278,4 +1278,4 @@ private function removeBlockingTimeOnUser(): void
         $unblockStatement->bindValue(':contactAlias', $this->login, \PDO::PARAM_STR);
         $unblockStatement->execute();
     }
-}
+}
```

Comment:
You can also remove this empty line to automatically exclude PHP developers from the review list.

Reply:
sorry 👍

Reply:
i appreciate a lot your understanding :open_hands:

Reply:
since i write in english i should learn to write in polite way :)   

This PR is currently in draft mode, so commenting on it at this stage may not be necessary. As per the established process, it requires approval from the Product Owner (PO) before I can proceed to revise the PR. Once that is done, I will be happy to open it up for reviewers. Since the development is still ongoing, there is a possibility that some elements might be removed. In my opinion, it would be beneficial for a PHP file to be reviewed by the PHP team as it provides an opportunity to explain the reasons behind the changes made, considering that the merge process is automated.